portfolio/src/main/webapp/backpacking.html

```diff
@@ -6,19 +6,139 @@
+      </div>
+      <div class="mySlides">
+        <div class="numbertext">9 / 10</div>
+          <img src="images/backpacking/bp-10.jpg" style="width:100%">
+      </div>
+      <div class="mySlides">
+        <div class="numbertext">10 / 10</div>
+          <img src="images/backpacking/bp-7.jpg" style="width:100%">
+      </div>
+
+      <!-- Next and previous buttons -->
+      <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+      <a class="next" onclick="plusSlides(1)">&#10095;</a>
+
+      <!-- Image text -->
+      <div class="caption-container">
+        <p id="caption"></p>
+      </div>
+
+      <div class="row">
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-1.JPG" 
+          style="width:100%" onclick="currentSlide(1)" 
+          alt="Lake Ediza, Ritter and Banner Peaks; John Muir Trail">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-2.JPG" 
+          style="width:100%" onclick="currentSlide(2)" 
+          alt="Ritter and Banner Twin Peaks; John Muir Trail">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-4.JPG" 
+          style="width:100%" onclick="currentSlide(3)" 
+          alt="Muir Pass; John Muir Trail">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-5.JPG" 
+          style="width:100%" onclick="currentSlide(4)" 
+          alt="Ascent to Dragon's Tooth; Virginia Triple Crown">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-6.JPG" 
+          style="width:100%" onclick="currentSlide(5)" 
+          alt="Paradise, Rainier; Mt. Rainier National Park">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-9.jpg" 
+          style="width:100%" onclick="currentSlide(6)" 
+          alt="Hoh Rainforest; Olympic Peninsula">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-8.jpg" 
+          style="width:100%" onclick="currentSlide(7)" 
+          alt="Hoh Rainforest; Olympic Peninsula">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-10.jpg" 
+          style="width:100%" onclick="currentSlide(8)" 
+          alt="Hole in the Wall, Rialto Beach; Shores of the Olympic">
+        </div>
+        <div class="column">
+          <img class="demo cursor" src="images/backpacking/bp-7.jpg" 
+          style="width:100%" onclick="currentSlide(9)" 
+          alt="Hole in the Wall, Rialto Beach; Shores of the Olympic">
+        </div>
       </div>
     </div>
+
+    <script>
+      var slideIndex = 1;
+      showSlides(slideIndex);
+
+      function plusSlides(n) {
+        showSlides(slideIndex += n);
+      }
+
+      function currentSlide(n) {
```

Comment:
`currentSlide` is a little vague because it doesn't tell you what action this function is taking. Consider something like `setCurrentSlide` or `changeSlideTo` instead